services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/attachment/application/ApplicationAttachmentService.kt

```diff
@@ -72,7 +77,11 @@ class ApplicationAttachmentService(
                 application = application,
             )
 
-        return attachmentRepository.save(applicationAttachment).toMetadata().also {
+        val savedAttachment = attachmentRepository.save(applicationAttachment)
```

Comment:
Is refusing to save an attachment after the application is no longer pending coming in a later PR or should be added in this one?